src/lib/holocene/select/native-select.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+  export let label = '';
+  export let id: string;
+  export let placeholder: string;
+  export let value = '';
+  export let options: { label: string; value: string }[] = [];
+</script>
+
+{#if label}
+  <label for={id} class={$$props.labelClass}>{label}</label>
```

Comment:
If the label is wrapped around the select you can skip the ID because it's associated and majestical